cpu/esp32/irq_arch.c

```diff
@@ -150,6 +159,8 @@ esp_err_t esp_intr_disable(intr_handle_t handle)
 esp_err_t esp_intr_alloc(int source, int flags, intr_handler_t handler,
                          void *arg, intr_handle_t *ret_handle)
 {
+    DEBUG("%s source=%d flags=0x%04"PRIx16" handler=%p arg=%p\n",
```

Comment:
Is `PRRx16` actually supported? I recall that newlib's nano stdio doesn't support `%hu` / `%hx` as well ass `%llu` and `%llx`.

If it works, perfect. Otherwise maybe `%x` and casting to `unsigned`?

Reply:
It seems to work.
```
esp_intr_alloc source=59 flags=0x0c02 handler=0x40375f7c arg=0
```